download/index.html

```diff
@@ -0,0 +1,130 @@
+---
+layout: page_column
+title: Portmaster Download
+heading: Portmaster Download
+exclude_kickstarter_plug: true
+---
+
+<div class="ui placeholder segment" style="display: block;text-align: center;padding: 1em 2.5em;">
+  <span class="ui red text" style="font-size: 1.15em;"><b>Warning:</b></span> The Portmaster is still in its early "pre-alpha" development stage. It is functional, but has not yet been tested widely. Thus, currently <b>we expect you to</b> have the technical knowledge to troubleshoot and <b>fix issues on your end by yourself.</b> Limited support is available, see below.
+</div>
+<div class="ui basic very padded segment">
+  <div class="ui three column centered doubling stackable grid">
+    <div class="four wide column center aligned">
+      <a href="https://updates.safing.io/latest/windows_amd64/packages/portmaster-installer.exe" target="_blank" class="ui windows button">
+        <i class="white windows button icon"></i>
+        Windows&nbsp;10
+      </a>
+    </div>
+    <div class="four wide column center aligned">
+      <a href="https://updates.safing.io/latest/linux_amd64/packages/portmaster-installer.deb" target="_blank" class="ui ubuntu button">
+        <i class="white ubuntu button icon"></i>
+        Ubuntu&nbsp;<span class="ui small text">(latest)</span>
+      </a>
+    </div>
+    <div class="four wide column center aligned">
+      <button class="ui apple disabled button">
+        <i class="white apple button icon"></i>
+        MacOS&nbsp;<span class="ui small text">(expect&nbsp;~&nbsp;2021)</span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<h1 class="ui horizontal divider header">Support</h1>
+
+<div class="ui basic very padded segment">
+  <div class="ui doubling center aligned grid">
+    <div class="column">
+      <p>Please report technical issues on <a href="{{ site.github_url }}" target="_blank">Github</a>, and ask for help on <a href="{{ site.reddit_url }}" target="_blank">Reddit</a></p>
```

Comment:
Please refer to the `portmaster` and `portmaster-ui` repos respectively. If you think this is confusing, we'll have to figure something out.